packages/components/NEW-COMPONENT-CHECKLIST.md

```diff
@@ -5,13 +5,55 @@ This is the quest issue/engineering checklist for the COMPONENT_NAME Component.
 ## Pre-Flight Checklist
 Update this list and these links as appropriate.
 
-- [Component Requirement Document (CRD)](url_here): This file should exist before any code is written.
-- [FIGMA Design](url_here): Since we are trying to align the component API naming with the same terms used in the Figma file, it is likely useful to have a fairly stable Figma design before we create a component; it should definitely be finalized before the component ships, however.
+- [Component Audit](https://www.figma.com/file/h3LtloARmbYVwh7z6XWHVb/Component-Audits-FY23?node-id=0%3A1): This audit should be completed by ambassadors before any work gets started.
+- [Component Requirement Document (CRD)](url_here): This file should exist before any design explorations or code is written.
 - [Design System Website](https://design-system-website.vercel.app/?path=/story/example-introduction--page) (storybook of storybooks): use for reference, to consider existing features that we might need to replicate in the component.
 
+
+## Design Checklist 
+The design checklist has 5 parts: creating the branch, design explorations and iterations, documentation, figma release, and testing. The design process is largely cyclical, so some of these steps may need to be repeated until approval.
+
+### Component Creation
+- [ ] [create a branch](https://help.figma.com/hc/en-us/articles/360063144053-Create-branches-and-merge-changes#Create_a_branch) in the [product components ui kit](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components)
+- [ ] add a new page for the component, as needed
+- [ ] **design**
+  - [ ] work through design explorations, providing 2-3 visual directions based on existing [foundations](https://www.figma.com/file/oQsMzMMnynfPWpMEt91OpH/HDS-Product---Foundations?node-id=2916%3A4) and patterns, as well as the findings from the audit and requirements from the CRD
+  - [ ] review with appropriate stakeholders (typically HDS designers, engineering partner, ambassadors) and iterate, as necessary
+  - [ ] review for a11y considerations and iterate, as necessary
+  - [ ] review with design leadership and iterate, as necessary
+  - [ ] once approved, build out the design as a component
+    - ensure the variant and property names align with what was discussed/decided on with your engineering partner
+- [ ] **documentation**
+  - [ ] stickersheet
+    - make sure the component instances can be easily selected and other layers are locked
+  - [ ] design guidelines
+    - when complete, lock this frame
+  - [ ] figma tips, as necessary
+    - when complete, lock this frame
+  - [ ] migration details, as necessary
+    - when complete, lock this frame
+- [ ] **release**
```

Comment:
I'm not sure this would fit in here or in the **design** section, but we are missing something like “_Ask the team lead for final review or approval on the component_”, even if it’s just a thumbs up from @didoo in this case.